scripts/rebuild_glyphset_presets.py

```diff
@@ -160,6 +161,18 @@ def getKoeberlinLatinGlyphSets():
             }
         )
 
+    order = ["XS", "S", "M", "L", "XL", "XXL"]
```

Comment:
```python
order = {k: i for i, k in enumerate(["XS", "S", "M", "L", "XL", "XXL"])}
```
Then use `order.get(glyphSet["name"].split()[-1], len(order))` (not `len(order) - 1`)

This avoids the ugly and unnecessary duplication of `glyphSet["name"].split()[-1]`.

Reply:
Thanks. Done.